docs/APIRef.DeviceObjectAPI.md

```diff
@@ -261,7 +261,7 @@ await device.uninstallApp('other.bundle.id');
 ### `device.openURL({url, sourceApp[optional]})`
 Mock opening the app from URL. `sourceApp` is an optional **iOS-only** parameter to specify source application bundle id.
 
-Read more in [Mocking Open From URL](APIRef.MockingOpenFromURL.md) section.
+Read more in [Mocking Open With URL](APIRef.MockingOpenWithURL.md) section.
```

Comment:
Can you pls change to "with" instead of "With"? Thanks!

Reply:
```suggestion
Read more in [Mocking Open with URL](APIRef.MockingOpenWithURL.md) section.
```